patterns/design-patterns-behavioral/src/test/java/com/baeldung/nulls/UsingOptionalUnitTest.java

```diff
@@ -11,32 +11,45 @@
 
 class UsingOptionalUnitTest {
 
-    private UsingOptional classUnderTest;
+    private UsingOptional dataObject;
 
     @BeforeEach
     public void setup() {
-        classUnderTest = new UsingOptional();
+        dataObject = new UsingOptional();
     }
 
     @Test
     public void whenArgIsFalse_thenReturnEmptyResponse() {
 
-        Optional<Object> result = classUnderTest.process(false);
+        Optional<Object> result = dataObject.process(false);
 
         assertFalse(result.isPresent());
     }
 
     @Test
     public void whenArgIsTrue_thenReturnValidResponse() {
 
-        Optional<Object> result = classUnderTest.process(true);
+        Optional<Object> result = dataObject.process(true);
 
         assertTrue(result.isPresent());
     }
 
     @Test
     public void whenArgIsFalse_thenChainResponseAndThrowException() {
 
-        assertThrows(Exception.class, () -> classUnderTest.process(false).orElseThrow(() -> new Exception()));
+        assertThrows(Exception.class, () -> dataObject.process(false).orElseThrow(() -> new Exception()));
     }
+
+    @Test()
+    void givenEmptyList_whenFindFirst_getDefaultValue() {
+        assertTrue(dataObject.findFirst().equalsIgnoreCase(UsingOptional.DEFAULT_VALUE));
+
+    }
+
+    @Test()
+    void givenEmptyList_whenFindOptionalFirst_returnsEmptyOptional() {
+        assertFalse(dataObject.findOptionalFirst().isPresent());
+
```

Comment:
redundant blank line - please also note the other instances of that in this class - there's no need for a blank space just after or just before the `{` or `}` of a function.